src/transforms/operationTransforms.ts

```diff
@@ -291,13 +292,22 @@ export function transformOperationResponse(
 
   const isDefault = httpInfo.statusCodes.indexOf("default") > -1;
 
-  return {
-    statusCodes: httpInfo.statusCodes,
+  const resultResponse = {
     mediaType: httpInfo.knownMediaType,
     mappers,
     types,
     isError: isDefault || isError
   };
+
+  const hash = createHash("md5")
+    .update(JSON.stringify(resultResponse))
```

Comment:
I _think_ this should be fine...the only thing that has me slightly worried is if you had two JavaScript objects that had fields added to them in a different order, then the hash wouldn't match even if the contents were the same. I think a deep equality check (I think lodash has one) would cover that scenario and doesn't seem like it would be more expensive since we're only checking the hash in one place, but I'll leave it up to you (presumably our generated test clients should catch this scenario).

Reply:
Yeah, you're right. I didn't think about the order 😄. It is not impacting any of our current tests, however it would be a timebomb. I'll use lodash's isEqual instead, as you suggested. Thanks for catching this